subprojects/inline/src/test/java/org/mockitoinline/SuperCallTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2017 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockitoinline;
+
+import org.junit.Test;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+public final class SuperCallTest {
```

Comment:
Nice test coverage!

I'm wondering if we should rather put the inline tests in main mockito codebase. The use cases should work for every Mockito mock maker, inline or not. We do run all our tests with inline mock as far as I remember.

Reply:
Do we? I would not recall that. If so, yes, we should probably move the class.